api/src/main/java/io/opentelemetry/OpenTelemetry.java

```diff
@@ -128,14 +133,16 @@ public static void setPropagators(ContextPropagators propagators) {
 
   /** Lazy loads an instance. */
   private static OpenTelemetry getInstance() {
-    if (instance == null) {
-      synchronized (OpenTelemetry.class) {
-        if (instance == null) {
-          instance = new OpenTelemetry();
-        }
+    mutex.lock();
```

Comment:
Any reason to remove the 'pre-check' without the lock?

Reply:
unfortunately, errorprone will complain if you try to do this. I guess I could remove the @GuardedBy on the field?

Reply:
Another argument for the plain `Object` I guess.

Reply:
errorprone will do the same complaining in that case as well.

Reply:
Ah, I see it does not understand double-checked locking!